src/app/main.cpp

```diff
@@ -153,8 +144,16 @@ int main( int argc, char **argv )
 
   QTranslator qfieldTranslator;
   QTranslator qtTranslator;
-  qfieldTranslator.load( QLocale(), "qfield", "_", ":/" );
-  qtTranslator.load( QLocale(), "qt", "_", ":/" );
+  if ( !customLanguage.isEmpty() )
```

Comment:
What happens if the if body is never executed and call `app.installTranslator()` without loading a language?

Reply:
Well, if the custom language fails, we revert to what we had before. If the system locale doesn't have translations, it reverts to strings in source code.